core/src/style/polygonStyle.cpp

```diff
@@ -65,56 +96,114 @@ void PolygonStyle::constructShaderProgram() {
     m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
 }
 
-VboMesh* PolygonStyle::newMesh() const {
-    return new Mesh(m_vertexLayout, m_drawMode);
-}
+namespace {
```

Comment:
OK

I think it would be worth to consider using more namespaces in future - instead of using many inner classes for name spacing and having that long class names for internal classes.  like e.g. tangram::style::polyline::Builder